app/models/server/raw/Rooms.js

```diff
@@ -1,6 +1,8 @@
+import { MongoInternals } from 'meteor/mongo';
```

Comment:
we cannot add Meteor's dependency to raw collections :see_no_evil: 

I was looking into how we could get the DB instance without this, the only way I found was changing the `BaseRaw` constructor to receive the `db` instance instead of a collection instance, which is a huge change :eyes:

Reply:
Yeah, i tried with the other way of importing the normal model, but app complained 😬. What I was thinking is to not to use the `readSecondaryPreferred` function, since we now we won't use any tags or something. That way we won't have to rewrite the entire raw models codebase lol.

Reply:
I see.. we'll need to figure out a way to do this without importing `MongoInternals` from `meteor/mongo` or do not use `readSecondaryPreferred` function then =\